libc-test/build.rs

```diff
@@ -4473,7 +4474,11 @@ fn test_linux(target: &str) {
         // `anonymous_1` is an anonymous union
         (struct_ == "ptp_perout_request" && field == "anonymous_1") ||
         // `anonymous_2` is an anonymous union
-        (struct_ == "ptp_perout_request" && field == "anonymous_2")
+        (struct_ == "ptp_perout_request" && field == "anonymous_2") ||
+        // FIXME: `adjust_phase` requires >= 5.7 kernel headers
+        // FIXME: `max_phase_adj` requires >= 5.19 kernel headers
+        // the rsv field shrunk when those fields got added, so is omitted too
+        (struct_ == "ptp_clock_caps" && (musl || loongarch64 || sparc64) && (["adjust_phase", "max_phase_adj", "rsv"].contains(&field)))
```

Comment:
Musl kernel header updates are in the queue right now https://github.com/rust-lang/libc/pull/3921, mind just waiting until that lands then dropping the `musl`?

Reply:
sounds good, I'll squash and rebase in a bit then